File: make.py
domains = open("domains", "r").read().split('\n')
output = open('hostfile', "w")
output.write("##COINHIVE DOMAINS\n")
for domain in domains:
	output.write("0.0.0.0 " + domain + "\n")
output.write("##END COINHIVE\n")

File: make.js
const fs = require('fs')

const domains = fs.readFileSync('domains', 'utf8').split('\n')

fs.appendFile('hostfile', "##COINHIVE DOMAINS\n')
domains.forEach((domain) => {
  fs.appendFile('hostfile', '0.0.0.0 ' + domain + '\n')
})
fs.appendFile('hostfile', '##END COINHIVE\n')
